### ISSUE TITLE
Empty, zero-length test files

### ISSUE THREAD

**Issue by musicinmybrain** [mechmotum/cyipopt#135]
The following files in `cyipopt/tests` are zero-length—not only empty, but without the single newline that a proper empty text file would have.

I understand empty `__init__.py` files, but for the test files it seems like something is missing. Was there ever anything here? Should the empty `test_*.py` files be removed?

- `cyipopt/tests/__init__.py`
- `cyipopt/tests/integration/__init__.py`
- `cyipopt/tests/integration/test_lasso.py`
- `cyipopt/tests/integration/test_rosen.py`
- `cyipopt/tests/unit/__init__.py`
- `cyipopt/tests/unit/test_defaults.py`
- `cyipopt/tests/unit/test_errors.py`
- `cyipopt/tests/unit/test_exceptions.py`
- `cyipopt/tests/unit/test_ipopt_funcs.py`
- `cyipopt/tests/unit/test_options.py`

**Comment by brocksam** [comment]
These were placeholder files that I created when refactoring CyIpopt during the creation of v1.0.0. A large portion of CyIpopt remains formally untested. These were some areas I'd identified tests would be good to have.

> * `cyipopt/tests/__init__.py`
> * `cyipopt/tests/integration/__init__.py`
> * `cyipopt/tests/unit/__init__.py`

Correct, I think these were needed to get Pytest to work properly (although I may be mistaken here).
 
> * `cyipopt/tests/integration/test_lasso.py`
> * `cyipopt/tests/integration/test_rosen.py`

The v0.x "tests" included the Lasso and Rosen problems. These were moved to `examples/` in v1.0, but would probably be good if these were included in the corresponding files as further integration tests, perhaps following `cyipopt/tests/integration/hs071.py`.

> * `cyipopt/tests/unit/test_defaults.py`
> * `cyipopt/tests/unit/test_errors.py`
> * `cyipopt/tests/unit/test_exceptions.py`
> * `cyipopt/tests/unit/test_ipopt_funcs.py`
> * `cyipopt/tests/unit/test_options.py`

These were just some other areas where I thought we needed some formally testing to ensure the package actually functions the way the we expect. And will be very valuable going forward as it would help avoid future PRs accidentally introducing unexpected bugs.

If this is something you'd be interested in contributing, it would be greatly appreciated :)
